### PR TITLE
IEP-862 Repeatedly ask to install some missing tools

### PR DESCRIPTION
**Pull request by sigmaaa**
## Description

Added an additional check, if founded versions are 3.6 and higher

Fixes # ([IEP-862](https://jira.espressif.com:8443/browse/IEP-EP-862))

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## How has this been tested?

Test 1:
- Install multiple versions of python on Windows so that these versions are in the Windows registry
- Try installing tools from eclipse. 
- List install tools -> no repeatedly ask to install missing tools

**Test Configuration**:
* ESP-IDF Version:
* OS (Windows):

## Dependent components impacted by this PR:

- Install Tools command
- List Installed Tools command

## Checklist
- [x] PR Self Reviewed
- [x] Applied Code formatting
- [ ] Added Documentation
- [ ] Added Unit Test
- [ ] Verified on all platforms - Windows, Linux and macOS

### CODE REVIEW

File: bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/update/AbstractToolsHandler.java [hunk 1]
```diff
@@ -120,11 +123,10 @@ protected String getPythonExecutablePath()
 			{
 				Logger.log("No Python installations found in the system."); //$NON-NLS-1$
 			}
```
Comment (kolipakakondal) [inline bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/update/AbstractToolsHandler.java:125]: Can we return here itself if there are no python versions found? https://github.com/espressif/idf-eclipse-plugin/blob/2416d02286b3fea2022547f41aa06fb0387fcc45/bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/update/AbstractToolsHandler.java#L124

**Review by kolipakakondal — CHANGES_REQUESTED** [review]
LGTM

File: bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/update/AbstractToolsHandler.java (hunk 1 above)
Comment (sigmaaa) [reply bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/update/AbstractToolsHandler.java:125]: Hi @kolipakakondal, but if pythonVersions are empty, we are trying to get the python executable from IDFUtil::getPythonExecutable. Python can be installed on the windows machine without adding it to the windows registry, in this case, we will always return a null here, even if a user provided the python executable during tools installation.

Comment (kolipakakondal) [reply bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/update/AbstractToolsHandler.java:125]: Ok, got it.

**Comment by kolipakakondal** [comment]
Hi @AndriiFilippov Could you verify this?

**Review by kolipakakondal — APPROVED** [review]
LGTM

**Comment by AndriiFilippov** [comment]
@sigmaaa hi ! 

Tested under: 
OS - Windows 10 
ESP-IDF: master/v5.0

LGTM ! 

image

**Review by AndriiFilippov — APPROVED** [review]

**Pull request merged**